gst/edge/edge_sink.c

```diff
@@ -118,6 +120,17 @@ gst_edgesink_class_init (GstEdgeSinkClass * klass)
           "The main topic of the host and option if necessary. "
           "(topic)/(optional topic for main topic).", "",
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_WAIT_CONNECTION,
+      g_param_spec_boolean ("wait-connection", "Wait connection to edgesrc",
+          "Wait until edgesink is connected to edgesrc. "
+          "In case of false(default), the buffers entering the edgesink are dropped.",
+          FALSE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_CONNECTION_TIMEOUT,
+      g_param_spec_uint64 ("connection-timeout",
+          "Timeout for wating a connection",
+          "The timeout (in microseconds) for wating a connection to recevier. "
```

Comment:
How about fixing typos?   `waiting` and `receiver`.

Reply:
Fixed. Thanks.